packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -1020,9 +1020,9 @@ const runGenerator = async (
             ...(selectedAppExtensions.length > 0 && {
                 mobify: {
                     app: {
-                        extensions: selectedAppExtensions.map((appExtensionName) => [
```

Comment:
Unrelated change to the PR that changes the generator to use strings instead of tuples inside the mobify in `package.json` file to declare the extensions without configuration.